app-android/src/main/res/mipmap-anydpi-v26/ic_launcher.xml

```diff
@@ -2,4 +2,5 @@
 <adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
     <background android:drawable="@drawable/ic_launcher_background"/>
     <foreground android:drawable="@drawable/ic_launcher_foreground"/>
-</adaptive-icon>
\ No newline at end of file
+    <monochrome android:drawable="@drawable/ic_launcher_foreground" />
```

Comment:
At first, I downloaded [svg image](https://www.figma.com/file/MbElhCEnjqnuodmvwabh9K/DroidKaigi-2023-App-UI?type=design&node-id=57713-28823&mode=design&t=iH9C6vVMHbFFKiMf-0) from Figma.
But, However, the output XML data was quite different from the existing ic_launcher_foreground.xml, so the icon size became larger than expected.

If you are fine with the results shown in the screenshot, we can reuse ic_launcher_foreground.
This is based on the response last year. https://github.com/DroidKaigi/conference-app-2022/pull/321

Reply:
Let me consult with the designer